src/plugins/workspace/public/workspace_client.ts

```diff
@@ -204,6 +204,7 @@ export class WorkspaceClient implements IWorkspaceClient {
     settings: {
       dataSources?: string[];
       permissions?: SavedObjectPermissions;
+      dataConnections?: string[];
```

Comment:
Why not reuse `dataSources`? It doesn't seem necessary to add an extra parameter `dataConnections`. Because for workspace, they are all data sources, I personally don't want it to be implemented in a way that every time adding new type of data source, it requires to change the API interface.

Reply:
what's passed in here is just saved object ids, we could handle different type of data source internally.

Reply:
Yes, if we reuse dataSources field, we need to update this field from [‘id1’,‘id2’] to [{type:‘data-source’,id:‘id1’},{type:‘data-connection’,id:‘id2’], which would make a breaking change to existing API.